src/cur-mot/add-who.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+"""
+Add `who=` attribute to signators.
+"""
+from common.xml_utils import write_xml
+from glob import glob
+from lxml import etree
+from pyriksdagen.utils import (
+    parse_protocol
+)
+from tqdm import tqdm
+import argparse, os, sys
+
+
+
+
+def get_person_D(root, ns):
+    D = {}
+    person_list = root.find(f".//{ns['tei_ns']}listPerson")
+    for p in person_list:
+        try:
+            _name = p.find(f"{ns['tei_ns']}name").text.strip()
+        except:
+            _name = None
+        try:
+            idno = p.find(f"{ns['tei_ns']}idno").text.strip()
+        except:
+            idno = None
+        if idno is not None and _name is not None:
+            D[idno] = _name
+    return D
+
+
+def match_ids(root, ns, person_D, unmatched, start, end):
+    try:
+        sig_list = root.find(f".//{ns['tei_ns']}div[@type='motSignatures']/{ns['tei_ns']}list")
+        assert sig_list is not None
+    except:
+        try:
+            sig_list = root.find(f".//{ns['tei_ns']}div[@type='motSignatures']/list")
+            assert sig_list is not Non
+        except:
+            sig_list = root.find(f".//div[@type='motSignatures']/list")
+
+    if sig_list is None:
+        print(root.find(f".//{ns['tei_ns']}div[@type='motBody']"))
+        utgone = False
+        try:
+            body = root.find(f".//{ns['tei_ns']}div[@type='motBody']").itertext()
+        except:
+            body = None
+        if body is not None:
+            for txt in body:
+                print('-', txt.strip())
+                if "Motionen utgår" in txt.strip():
+                    print("x", body)
+                    utgone = True
+            if utgone == False:
+                xmlns = ns['xml_ns']
+                with open(f"riksdagen-motions/_no-sig-block_{start}-{end}.txt", "a+") as log:
```

Comment:
Could we use the [logging module](https://docs.python.org/3/library/logging.html)?